tedge_config/src/models/file_path.rs

```diff
@@ -22,12 +22,6 @@ where
     }
 }
 
-impl Into<PathBuf> for FilePath {
-    fn into(self) -> PathBuf {
-        self.0
-    }
-}
```

Comment:
It kind of because a couple lines above we have generic `impl From`:
```rust
impl<T> From<T> for FilePath
where
    PathBuf: From<T>,
{
    fn from(input: T) -> Self {
        Self(PathBuf::from(input))
    }
}
```